### PR TITLE
162830 IIB Deploy step hangs if no response is received

### PR DESCRIPTION
**Pull request by nhmathis**
The Deploy Step now uses the timeout to wait. After half the time period has passed, it will poll the response every 10 seconds until the complete timeout has elapsed or a response has been received. 

The package was missing from IIB9ConnectionBroker.groovy

### CODE REVIEW

**Comment by nhmathis** [comment]
While testing I found that the disconnectAll() method is only present in v10, not v9. To preserve full compatibility, I've changed it back to disconnect(). I've confirmed with jrbarto that is the appropriate path to take.

**Review by jrbarto — APPROVED** [review]

File: src/main/groovy/com/urbancode/air/plugin/wmbcmp/IIBHelper.groovy [hunk 1]
```diff
@@ -52,7 +53,8 @@ class IIBHelper {
         version = integerPoint != -1 ? version.substring(0, integerPoint) : version
         def versionInt = version.toInteger()
 
-        timeout = Integer.valueOf(props['timeout']?.trim()?:-1) // time to wait for broker response
+        Integer tempTimeout = Integer.valueOf(props['timeout']?.trim())
+        timeout = tempTimeout > -1 ? tempTimeout : -1) // time to wait for broker response
```
Comment (jrbarto) [inline src/main/groovy/com/urbancode/air/plugin/wmbcmp/IIBHelper.groovy]: ending ) breaks the plugin here.

**Comment by nhmathis** [comment]
Closing. Invalid implementation.

**Comment by nhmathis** [comment]
Invalid approach. Closing.

**Pull request closed without merging**